erpnext/stock/doctype/stock_entry/stock_entry.py

```diff
@@ -691,6 +691,7 @@ def update_stock_ledger(self):
 		self.make_sl_entries(sl_entries, self.amended_from and 'Yes' or 'No')
 
 	def update_batch_with_customer_provided_item(self):
+		# update batch doc with provided by customer if batch doc don't have provided by customer
```

Comment:
Same case here, comment is useless.

Reply:
Specify that is is being done only when the stock entry is a "Material Receipt". Each of the "if" statements should have a comment before explaining the purpose of the cases. The logic of the function should be visible in the comments.